### PR TITLE
Change ClusterOperatorStatus Failed to Degraded

### PR DESCRIPTION
**Pull request by eparis**
This is just a rename, but it should take on a slightly newer definition over time.

https://github.com/openshift/cluster-version-operator/pull/161

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package: github.com/openshift/library-go
 import:
 - package: k8s.io/apimachinery
+  version: kubernetes-1.13.4
```
Comment (eparis) [inline glide.yaml]: how the heck was this not already pinned? do you know how long it took me to figure out this was my problem? I've gotten so dumb.

**Comment by eparis** [comment]
based on https://github.com/openshift/api/pull/287

File: glide.yaml (hunk 1 above)
Comment (deads2k) [reply glide.yaml]: > how the heck was this not already pinned? do you know how long it took me to figure out this was my problem? I've gotten so dumb.

Clearly that guy should be fired.

**Comment by deads2k** [comment]
everybody jump!

/lgtm

**Comment by eparis** [comment]
/hold
probably shouldn't jump until:
https://github.com/openshift/cluster-version-operator/pull/165

**Comment by eparis** [comment]
see: https://github.com/openshift/api/pull/287

**Comment by deads2k** [comment]
/hold cancel

running for API freeze, the CVO will take the update eventually, releasing this

**Pull request merged**